.gitignore

```diff
@@ -34,6 +34,7 @@ wp-config.php
 /wp-content/uploads/*
 /wp-content/mu-plugins/load-wp-performance-profiler.php
 /wp-content/mu-plugins/wp-performance-profiler/*
+test.html
```

Comment:
Should this be here?

Reply:
Yes, I use this to test out various markups locally but don't want it included in the main repo.